### PR TITLE
Update to v3.0.4 and document local differences

### PR DESCRIPTION
**Pull request by snowak-cb**
- Update CB fork of `stacktrace-gps` to match [upstream `3.0.4`](https://github.com/stacktracejs/stacktrace-gps/tree/v3.0.4) 
- Document CB-specific changes 
- Build & commit `dist` directory following upstream's process

### CODE REVIEW

File: dist/stacktrace-gps.js [hunk 1]
```diff
@@ -129,12 +129,16 @@
...
+        // eslint-disable-next-line no-cond-assign
+        while (matchSourceMappingUrl = sourceMappingUrlRegExp.exec(source)) {
             lastSourceMappingUrl = matchSourceMappingUrl[1];
         }
         if (lastSourceMappingUrl) {
             return lastSourceMappingUrl;
         } else {
+            // in upstream, this else block just throws an error 
+            // our prod webpack config uses `devtool: 'hidden-source-map'`, so prod error logs will never find a regex match
+            // instead, since we know the pattern that our maps follow, we'll assemble the path to our maps from the filename here
```
Comment (ghost) [inline dist/stacktrace-gps.js:141]: 🏆

File: stacktrace-gps.js [hunk 2]
```diff
@@ -129,12 +129,16 @@
...
+        // eslint-disable-next-line no-cond-assign
+        while (matchSourceMappingUrl = sourceMappingUrlRegExp.exec(source)) {
             lastSourceMappingUrl = matchSourceMappingUrl[1];
         }
         if (lastSourceMappingUrl) {
             return lastSourceMappingUrl;
         } else {
+            // in upstream, this else block just throws an error 
+            // our prod webpack config uses `devtool: 'hidden-source-map'`, so prod error logs will never find a regex match
+            // instead, since we know the pattern that our maps follow, we'll assemble the path to our maps from the filename here
```
Comment (ghost) [inline stacktrace-gps.js:141]: nicely explained! thanks!

**Comment by ghost** [comment]
Travis-CI PR is failing

**Comment by snowak-cb** [comment]
> Travis-CI PR is failing

Yeah that's expected, we have the upstream travis config but we don't have the right secrets/plugins configured here 🤷

**Pull request merged**